src/azure-cli/azure/cli/command_modules/appconfig/_format.py

```diff
@@ -174,6 +185,23 @@ def _get_value(item, *args):
         return ' '
 
 
+def _get_value_by_names(item, property_names):
+    """
+        CLI core converts KeyValue object field names to camelCase (eg: content_type becomes contentType)
+        But when customers specify field filters, we return a dict of requested fields instead of KeyValue object
+        In that case, field names are not converted to camelCase. We need to check for both content_type and contentType
+        We iterate through given names and return the first value that is found
+    """
+    empty_value = ' '
```

Comment:
Is empty_value a single space? Can you directly check for empty string in line 200?

Reply:
Yes, that is possible. It just wasn't easily apparent why I was using a single space so the variable was supposed to be self-explanatory. Would you suggest we do a direct comparison?

Reply:
Oh I see we always compared it with single space. I dont remember if that's intentional though. Could you check if it works with empty string instead of single space?

Reply:
I tested and did not see any issue using an empty string, however, the default [build_table_output](https://github.com/Azure/azure-cli/blob/fdb9aa742b404a433f207b573eea37341769020c/src/azure-cli-core/azure/cli/core/commands/transform.py#L98C1-L98C30) method uses empty spaces for null values. 
I am thinking it would be good to be consistent with this in case I might be missing something.

Reply:
Thanks for confirming! Yes, lets keep it as single space and add a comment explaining why its not an empty string.

Reply:
This has been updated, thanks!